### DISCUSSION TITLE
Barcamp: Usability Paginierungseditor

### DISCUSSION THREAD

**Discussion by SonjaGellert** [kitodo/kitodo-production#6359] [Ideas]
**Große Design-Veränderung:**
-	Seitenauswahl entfernen (Seitenauswahl über Strukturbaum oder Galerie)
-	Felder neu und dynamisch anordnen lassen, Textfelder nicht über die ganze Breite gehen lassen.

_Grafisches Beispiel:_
![Paginierung-responsive](https://github.com/user-attachments/assets/7f2d18b5-2c96-47e4-aaea-dd898ec32b74)

**Weitere Einstellungen:**
-	Logische Paginierung in der Galerie sichtbar machen. 
-	Weitere Paginierungstypen hinzufügen (bspw. alphabethisch)
-	Paginierung eines ausgewählten Strukturelements (alle Elemente die bspw. einem Kapitel zu geordnet sind)

**Einstellungen in den Benutzerprofilen erweitern:**
-	Bevorzugter (voreingestellter) Paginierungstyp auswählbar machen (Paginierung & Zählung)

**Kleine Veränderungen** (evtl. kurzfristig implimentierbar)
-	Doku zur Funktion für Fortgeschrittene in Oberfläche einfügen ([Wiki Link](https://github.com/kitodo/kitodo-production/wiki/Paginieren#paginierung-f%C3%BCr-fortgeschrittene)) vllt. Doku auf Kitodo 3 anpassen
-	Button "Abbrechen" umbenennen in "Zurücksetzen" (Verständlicher, was passiert wenn man drauf klickt)
-	Recto/Verso bei Blattzählung vertauschbar machen.
-	Intelligente Matrix bei Auswahlfeldern: (bei uncounted einige Einstellungen wie Paginierungsstartwert obsolet)

**Große Vision vllt für die Zukunft.**
"schwebende" skalierbare Popups aller Metadatenmodule zum selbst anordnen.

**Comment by subhhwendt** [comment]
Hier kommt zur Vervollständigung unsere kl. Protokollierung am Flipchart in Marburg
![20241119_130600](https://github.com/user-attachments/assets/60bdc280-3319-4fbc-acf2-5e80d388be5a)
![20241119_130607](https://github.com/user-attachments/assets/52ffb7ae-eb09-41be-8905-ad0506da7566)
![20241119_130614](https://github.com/user-attachments/assets/24b7c49d-7b40-480f-986f-5f17bed3f91a)
![20241119_130621](https://github.com/user-attachments/assets/e2159a59-af04-43b6-9a89-71ce9aec3ae5)
